src/Components/Web.JS/src/Rendering/LogicalElements.ts

```diff
@@ -55,8 +55,10 @@ export function toLogicalRootCommentElement(start: Comment, end: Comment): Logic
   const children = getLogicalChildrenArray(parentLogicalElement);
   Array.from(parent.childNodes).forEach(n => children.push(n as unknown as LogicalElement));
   start[logicalParentPropname] = parentLogicalElement;
-  start[logicalEndSiblingPropname] = end;
-  toLogicalElement(end, /* allowExistingcontents */ true);
+  if (end) {
```

Comment:
What does this change do? Does this need a comment explaining when it occurs?

Reply:
We didn't clean this up when we removed the prerendering support. This is only adding support for non-prerendered server components that only have a start comment tag (No end is necessary as they don't wrap any prerendered content)

Reply:
Added a comment.